model/app.py

```diff
@@ -561,6 +574,13 @@ def apply_additions(model, changes):
     return model
 
 
+def apply_measurements(model, changes):
+    for rxn in changes['reactions']:
+        model.reactions.get_by_id(rxn['id']).lower_bound = rxn['lower_bound']
```

Comment:
That function is removed in devel06 but alright, let's use here make it easy to remember to change when migrating.